lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1260,18 +1262,35 @@ def create_ReAuthRequest(self, imsi, policy_id, flow_list, qos):
         qos = QoSInformation(qci=qos["qci"])
 
         # Get sessionid
-        req = GetDirectoryFieldRequest(id=imsi, field_key="session_id")
-        try:
-            res = self._directorydstub.GetDirectoryField(
-                req, DEFAULT_GRPC_TIMEOUT
-            )
-        except grpc.RpcError as err:
-            logging.error(
-                "GetDirectoryFieldRequest error for id: %s! [%s] %s",
-                imsi,
-                err.code(),
-                err.details(),
-            )
+        #TODO: remove retries
+        i = 0
+        MAX = 3
+        res = None
+        while i < MAX:
+            req = GetDirectoryFieldRequest(id=imsi, field_key="session_id")
+            try:
+                res = self._directorydstub.GetDirectoryField(
+                    req, DEFAULT_GRPC_TIMEOUT
+                )
+            except grpc.RpcError as err:
+                print(
+                    "error: GetDirectoryFieldRequest error for id: %s! [%s] %s",
+                    imsi,
+                    err.code(),
+                    err.details(),
+                )
+            if req != None:
+                i = MAX
+            else:
+                i+=1
+                print("warning: directoryd failed to return sessionId for %s. Retrying", imsi)
+                time.sleep(3)
```

Comment:
@uri200 , I believe @ulaskozat is suggesting the following change, which simplifies the indentation. It does not change the logic, but makes it more readable

```suggestion
            if req != None:
               break
           
             i+=1
             print("Warning: directoryd failed to return sessionId for %s. Retrying", imsi)
             time.sleep(3)
```